samples/TEE_aware/gencreds/Makefile

```diff
@@ -0,0 +1,38 @@
+CFLAGS = -fPIC
+
+.PHONY: build run clean
+
+all: build
+
+build:  ext2rootfs package
+
+appdir:
+	echo "Building an appdir with all the necessary application files"
+	@rm -rf appdir
+	@mkdir -p appdir/bin
+	gcc $(CFLAGS) -I$(MYSTIKOS_INSTALL_DIR)/include gencreds.c -o appdir/bin/gencreds
+
+ext2rootfs: appdir
+	echo "Building a ext2 file system to run in Mystikos"
+	@myst mkext2 appdir ext2rootfs
+
+runexec: ext2rootfs
+	echo "Running application in ext2 file system with myst exec-sgx"
+	@myst exec-sgx ext2rootfs /bin/gencreds $(OPTS)
```

Comment:
Same suggestion as `TEE_aware/dotNET`. Use ext2 without root hash should be only for debugging.

Reply:
Added roothash